Vagrantfile

```diff
@@ -12,7 +12,7 @@ Vagrant.configure('2') do |config|
   config.disksize.size = '50GB'
   config.vm.box_check_update = false
   config.vm.host_name = 'sumologic-kubernetes-collection'
-  config.vm.network :private_network, ip: "192.168.78.66"
+  config.vm.network :private_network, ip: "192.168.56.2"
```

Comment:
It won't be as easy as 192.168.SU.MO anymore 😿 

Do you mind adding a comment above this line with e.g. a link to an issue or the note about the limitation?

Reply:
I don't think a comment is needed - `git blame` will tell you the story, right?

Reply:
I don't usually look at `git blame` output, do you? :) BTW, any change (even a whitespace change) within this line will "stack on top of this change" and it will be even harder to figure out why this was changed to be `... 56.2`

Reply:
I do 😛

Reply:
Added a comment, agreed that it should be there in case someone decides to change the IP in the future, especially since the effects aren't immediately obvious.